server/src/userManagement.mjs

```diff
@@ -44,6 +44,14 @@ function getUsers() {
 function isKnownUser(userId) {
   return user2wss.has(''+userId);
 }
+/* 
+* @function:parseUser()
+* @Description: parsing the userId to get user,group and appId
+* @Params:userId
+* @Return: Output object
+* @Example Input: 123~A#youtube
```

Comment:
The appIds need to be generic. Can this be "appId1" instead?

Also thank you for adding this. :)

Reply:
Done.

Reply:
I meant the "#youtube" in the example needs to become "#appId1. We don't need to change the variable name since it's already generic.

Reply:
Done